src/apps/search-result/search-result.tsx

```diff
@@ -29,13 +29,17 @@ import useGetCleanBranches from "../../core/utils/branches";
 import useFilterHandler from "./useFilterHandler";
 import SearchResultSkeleton from "./search-result-skeleton";
 import SearchResultZeroHits from "./search-result-zero-hits";
+import SearchResultInvalidSearch from "./search-result-not-valid-search";
+import { shouldShowInvalidSearchPage } from "./helper";
 
 interface SearchResultProps {
   q: string;
   pageSize: number;
 }
 
 const SearchResult: React.FC<SearchResultProps> = ({ q, pageSize }) => {
+  const queryParametersFromUrl = new URLSearchParams(window.location.search);
+  const qUrlParameter = queryParametersFromUrl.get("q");
```

Comment:
This is not necessary.
The `q` parameter that is being passed to this component comes from `SearchResultEntry` that uses `getParams` which is also written in the commentary [here](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/373a04708072387274812a8bbbdda9a4a6521851/src/apps/search-result/search-result.entry.tsx#L67-L69).

Reply:
That's good and well, but we still need to somehow figure out whether there is anything in the search bar or not in order to determine whether the user is searching for the string "null" or whether there is no q parameter there at all (which it should always be present when on the search result page - bc the autosuggest and search bar redirect to `/search?q={searchTerm}`

Reply:
yeah it makes sense together with your other comment :) I made it happen